fluss-rpc/src/main/java/com/alibaba/fluss/rpc/netty/client/NettyClientHandler.java

```diff
@@ -166,7 +166,7 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         if (evt instanceof IdleStateEvent) {
             IdleStateEvent event = (IdleStateEvent) evt;
             if (event.state().equals(IdleState.ALL_IDLE)) {
-                LOG.warn("Connection {} is idle, closing...", ctx.channel().remoteAddress());
+                LOG.debug("Connection {} is idle, closing...", ctx.channel().remoteAddress());
```

Comment:
if (LOG.isDebugEnabled()) {
    
    LOG.debug("Connection {} is idle, closing...", ctx.channel().remoteAddress());
    
}

Reply:
done